packages/matchbox/src/components/TimelineTabs/useTabConstructor.tsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
 import { onKey, onKeys } from '../../helpers/keyEvents';
 
-function useTabConstructor({ tabs, initialIndex }) {
-  const tabRefs = React.useRef({ current: [] });
-  const focusRef = React.useRef();
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function useTabConstructor({ tabs, initialIndex }: { tabs: any[]; initialIndex: number }): {
```

Comment:
Not sure what this should be, might need to refactor a bit to fix this `any[]`

Reply:
Would `React.ComponentProps<typeof TimelineTabsTab>[]` work?

Reply:
It's technically a `ReactChild` that is returned from `React.Children.toArray` I think, but it breaks because of the `...tab.props,` later in the file